File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cs
public class Solution
{
    public int LongestConsecutive(int[] nums)
    {
        if (nums.Length == 0) return 0;
        HashSet<int> set = new HashSet<int>(nums);
        int longest = 0;
        foreach (int num in set)
        {
            if (!set.Contains(num - 1))
            {
                int current = num + 1;
                while (set.Contains(current)) { current++; }
                int length = current - num;
                if (length > longest) { longest = length; }
            }
        }
        return longest;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        rr={}
        rk=[]
        for i in range(len(nums)):
            if nums[i] in rr:
                rr[nums[i]]=1+rr.get(nums[i],0)
            else:
                rr[nums[i]]=1
        for k, v in rr.items():
            print(k,":",v)
            if (v > k): rk.append(k)
        return rk